pkgs/development/libraries/webrtc-audio-processing/default.nix

```
@@ -1,14 +1,17 @@
-{ lib, stdenv, fetchurl, darwin }:
+{ lib, stdenv, fetchurl, darwin, meson, abseil-cpp, ninja }:
```

```suggestion
{ lib, stdenv, fetchurl, darwin, meson, abseil-cpp, ninja, ApplicationServices }:
```

Please inherit ApplicationServices in top-level.